rust_dev_preview/README.md

```diff
@@ -18,6 +18,37 @@ These examples demonstrate how to perform several operations using the developer
 
 ## Running the code examples
 
+### Environment Variables
+
+Unless otherwise noted, the AWS SDK for Rust uses these same environment variables as the AWS Command Line Interface (AWS CLI) to load account credentials, specify the default region, and to set other configuration settings.
+See [Configuring the AWS CLI - Environment Variables](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-envvars.html#envvars-list) for more details on these environment variables.
+
+- AWS_ACCESS_KEY_ID specifies an AWS access key associated with an IAM user or role.
+- AWS_IAM_ROLE_SESSION_NAME specifies the name to attach to the role session.
+- AWS_MAX_ATTEMPTS specifies the maximum number of retry attempts for retriable SDK operations.
+- AWS_PROFILE s pecifies the name of the AWS CLI profile with the credentials and options to use. This can be the name of a profile stored in a credentials or config file, or the value default to use the default profile.
+- AWS_REGION (AWS_DEFAULT_REGION) specifies the AWS Region to send requests to.
+- AWS_RETRY_MODE specifies which retry mode AWS CLI uses. The AWS SDK for Rust only supports retry mode `legacy` (the default).
+- AWS_ROLE_ARN specifies the Amazon Resource Name (ARN) of an IAM role with a web identity provider that you want to use to run the AWS CLI commands.
+- AWS_SDK_UA_APP_ID
```

Comment:
This customizes the app name in the user agent header that gets sent with each request.